### ISSUE TITLE
Error in 2.8 branch

### ISSUE THREAD

**Issue by R-0ne** [joric/io_scene_b3d#4]
hi, the 2.8 branch is not functional, it has an error when exporting

```

location: <unknown location>:-1

AttributeError: 'ExportB3D' object has no attribute 'axis_forward'
 
    return super().__getattribute__(attr)

File "C:\Program Files\Blender Foundation\Blender\2.80\scripts\modules\bpy_types.py", line 674, in __getattribute__

      global_matrix = axis_conversion(to_forward=self.axis_forward,

File "C:\Program Files\Blender Foundation\Blender\2.80\scripts\addons\io_scene_b3d\__init__.py", line 129, in execute

Traceback (most recent call last):
```

**Comment by joric** [comment]
Yeah, sorry, still in progress. Pull requests welcome.

**Comment by R-0ne** [comment]
> Yeah, sorry, still in progress. Pull requests welcome.

If I had the skills that would be happy ... I'm a little lost in all the changes of the API 2.8

Do you think about working on it soon?

thank you for all your work in passing

**Comment by joric** [comment]
Updated import here 541a1b07  (see #2)

**Comment by R-0ne** [comment]
thank you very much @joric  , I had seen your work but it is the export to the B3d format that we need which is the only way to have entities animate in Minetest.

**Comment by joric** [comment]
fixed 2.8 export a little bit here a8b05dc3 and fixed uvs here 5c9d2564

**Comment by R-0ne** [comment]
Hello,

Everything seems to work for export and animation with bones.

Thank you very much for taking the time to update your work.

You can close this issue
